.drone.yml

```diff
@@ -193,87 +71,25 @@ steps:
       - npm -v
       - npm ci
 
-  - name: format
-    image: node:18
+  - name: Check prettier
+    image: node:18.19
     commands:
       - npm run check-format
 
-  # - name: check lint
-  #   image: node:18
-  #  commands:
-  #    - npm run lint
-
-  - name: build
-    image: node:18
-    commands:
-      - npm run build
-
-  - name: extract-version-from-package
-    image: node:18
-    # generates the .tags file for the docker plugin
-    # add tag latest for the rolling tag
+  - name: Check types
+    image: node:18.19
     commands:
-      - echo -n "latest,$(npm pkg get version | sed 's/"//g')" > .tags
-
-  - name: build docker-image
-    image: plugins/docker
-    pull: always
-    settings:
-      registry: docker-regis.iex.ec
-      repo: docker-regis.iex.ec/product/web3mail-usecase-demo
-      build_args:
-        - COMMIT=${DRONE_COMMIT}
-        - VERSION=prod
-      dockerfile: Dockerfile
-      username:
-        from_secret: nexus-user
-      password:
-        from_secret: nexus-password
-      ssh_agent_key:
-        from_secret: ssh-key-team-product-github
+      - npm run check-types
 
-  - name: transfer deployment files to server
-    image: appleboy/drone-scp
-    pull: always
-    settings:
-      host:
-        from_secret: production-host
-      username:
-        from_secret: ssh-user-team-product-server
-      key:
-        from_secret: ssh-key-team-product-server
-      port: 22
-      source:
-        - deployment/common
-        - deployment/production
-      target: /opt/web3mail-usecase-demo/
-      overwrite: true
-      strip_components: 2
-
-  - name: deploy to production
-    image: appleboy/drone-ssh
-    pull: always
+  - name: Vercel deploy to prod
+    image: node:18.19
     environment:
-      DOCKER_USERNAME:
-        from_secret: nexus-user
-      DOCKER_PASSWORD:
-        from_secret: nexus-password
-    settings:
-      host:
-        from_secret: production-host
-      username:
-        from_secret: ssh-user-team-product-server
-      key:
-        from_secret: ssh-key-team-product-server
-      port: 22
-      envs:
-        - DOCKER_USERNAME
-        - DOCKER_PASSWORD
-      script:
-        - cd /opt/web3mail-usecase-demo/
-        - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD docker-regis.iex.ec
-        - docker compose pull
-        - docker compose down --remove-orphans
-        - sleep 5
-        - docker compose up -d
-        - docker logout docker-regis.iex.ec
+      VERCEL_ORG_ID: team_qEUZmPwX3J0FFDAk23fSvIlq
+      VERCEL_PROJECT_ID: prj_Fp3PcYR1CrST2QwpLO2fEwJFXkdO
+      VERCEL_TOKEN:
+        from_secret: vercel-token
+    commands:
+      - npm install --global vercel@canary
+      - vercel pull --yes --environment=production --scope i-exec --token $VERCEL_TOKEN
+      - SET_SUBPATH_FOR_PROD=true VITE_APP_COMMIT=${DRONE_COMMIT} vercel build --prod --scope i-exec --token $VERCEL_TOKEN
```

Comment:
Note here: `SET_SUBPATH_FOR_PROD` env var.
→ All asset file paths will be prefixed with '/showcase'.